admin/database/database.go

```diff
@@ -265,6 +267,7 @@ type Project struct {
 	ProdSlots            int       `db:"prod_slots"`
 	ProdTTLSeconds       *int64    `db:"prod_ttl_seconds"`
 	ProdDeploymentID     *string   `db:"prod_deployment_id"`
+	Tags                 Tags      `db:"tags"`
```

Comment:
Try just `[]string` after adding the `NOT NULL` change (and resetting the dev postgres) to the migration as suggested in another comment.

If that still doesn't work, try `pgtype.Array[string]` instead. It should be possible to get rid of the custom `Tags` type.